SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosInfoSupport.java

```diff
@@ -27,17 +27,17 @@
 import io.dropwizard.lifecycle.Managed;
 
 @Singleton
-public class SingularityLogSupport implements Managed {
+public class SingularityMesosInfoSupport implements Managed {
```

Comment:
this class name seems a little vague -- perhaps `SingularityMesosExecutorInfoSupport` ?